Makefile

```
@@ -34,7 +34,57 @@ endif
 GO=go
 GOFLAGS=""
 
+# requires:
+#	AWS_SECRET		- Secret from AWS
+#	AWS_ACCESS		- Access key from AWS
+#	S3_REGION		- Region of the bucket used to store the model
+#	S3_ENDPOINT		- Endpint of the bucket
+#	REGISTRY_USERNAME	- quay.io username
+#	REGISTRY_PASSWORD	- quay.io password
+#	REGISTRY_CREDENTIALS	- quay.io robot secret file
+go-test-setup:
+ifndef AWS_SECRET
+	$(error AWS_SECRET is undefined)
+endif
+ifndef AWS_ACCESS
+	$(error AWS_ACCESS is undefined)
+endif
+ifndef S3_REGION
+	$(error S3_REGION is undefined)
+endif
+ifndef S3_ENDPOINT
+	$(error S3_ENDPOINT is undefined)
+endif
+ifndef REGISTRY_USERNAME
+	$(error REGISTRY_USERNAME is undefined)
+endif
+ifndef REGISTRY_PASSWORD
+	$(error REGISTRY_PASSWORD is undefined)
+endif
+ifndef REGISTRY_CREDENTIALS
+	$(error REGISTRY_CREDENTIALS is undefined)
+endif
+	oc new-project test-namespace
+	@cat pipelines/tekton/aiedge-e2e/templates/credentials-s3.secret.yaml.template | sed 's#{{ AWS_SECRET }}#${AWS_SECRET}#g' | sed 's#{{ AWS_ACCESS }}#${AWS_ACCESS}#g' | sed 's#{{ S3_REGION }}#${S3_REGION}#g' | sed 's#{{ S3_ENDPOINT }}#${S3_ENDPOINT}#g' | oc create -f -
+	@cat pipelines/tekton/aiedge-e2e/templates/credentials-image-registry.yaml.template | sed 's#{{ REGISTRY_USERNAME }}#${REGISTRY_USERNAME}#g' | sed 's#{{ REGISTRY_PASSWORD }}#${REGISTRY_PASSWORD}#g' | oc create -f -
+	oc create -f ${REGISTRY_CREDENTIALS}
+	oc secret link pipeline $(shell yq '.metadata.name' ${REGISTRY_CREDENTIALS})
+	oc create -f pipelines/tekton/aiedge-e2e/test-data/bike-rentals-test-data-cm.yaml
+
+# requires:
+#	S3_BUCKET		- Name of S3 bucket that has the model
+#	REGISTRY_REPO		- quay.io robot secret file
+go-test go-test-setup:
```

I believe prerequisite should be written like this:
```suggestion
go-test: go-test-setup
```